[2022-10-26 14:45:44,550] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag.also_run_this manual__2022-10-26T14:45:38.988431+00:00 [queued]>
[2022-10-26 14:45:44,559] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag.also_run_this manual__2022-10-26T14:45:38.988431+00:00 [queued]>
[2022-10-26 14:45:44,560] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 14:45:44,560] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-26 14:45:44,561] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-26 14:45:44,574] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): also_run_this> on 2022-10-26 14:45:38.988431+00:00
[2022-10-26 14:45:44,577] {standard_task_runner.py:52} INFO - Started process 5290 to run task
[2022-10-26 14:45:44,580] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dag', 'also_run_this', 'manual__2022-10-26T14:45:38.988431+00:00', '--job-id', '152', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpafqunvas', '--error-file', '/tmp/tmpp0ob_en_']
[2022-10-26 14:45:44,581] {standard_task_runner.py:80} INFO - Job 152: Subtask also_run_this
[2022-10-26 14:45:44,639] {task_command.py:369} INFO - Running <TaskInstance: dag.also_run_this manual__2022-10-26T14:45:38.988431+00:00 [running]> on host cf01c04ee00d
[2022-10-26 14:45:44,711] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag
AIRFLOW_CTX_TASK_ID=also_run_this
AIRFLOW_CTX_EXECUTION_DATE=2022-10-26T14:45:38.988431+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-26T14:45:38.988431+00:00
[2022-10-26 14:45:44,712] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-10-26 14:45:44,712] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls']
[2022-10-26 14:45:44,718] {subprocess.py:85} INFO - Output:
[2022-10-26 14:45:44,719] {subprocess.py:96} INFO - Command exited with return code 0
[2022-10-26 14:45:44,751] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dag, task_id=also_run_this, execution_date=20221026T144538, start_date=20221026T144544, end_date=20221026T144544
[2022-10-26 14:45:44,793] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-26 14:45:44,827] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
